portal/templates/portal_wrapper.html

```diff
@@ -79,6 +79,11 @@
                             <li><a href="{{PORTAL}}/scheduled_jobs">{{ _("Scheduled Jobs") }}</a></li>
                             <li><a href="{{PORTAL}}/settings">{{ _("Settings") }}</a></li>
                             {% endif %}
+                            <!-- fixme use the new 'analytics' intervention's 'link_url' -->
+                            <!-- fixme consider intervention.display_for_user(user) ? -->
```

Comment:
I would not suggest `display_for_user()` - too expensive, and designed for patients, for which this doesn't apply